desktop/menus/view-menu.js

```diff
@@ -106,6 +106,10 @@ const buildViewMenu = settings => {
             label: '&Dark',
             id: 'dark',
           },
+          {
+            label: '&System',
+            id: 'system',
+          },
```

Comment:
Just a thought, but I've seen the "system" setting at the top of the list in other apps - especially if it's the default

Reply:
Done